var/spack/repos/builtin/packages/gtkplus/package.py

```
@@ -36,7 +36,7 @@ class Gtkplus(MesonPackage):
         deprecated=True,
     )
 
-    variant("cups", default=False, description="enable cups support")
+    variant("cups", default=True, description="enable cups support")
```

Why this change? If another package requires `gtkplus+cups`, it should say that.

---

In my testing, ```spack install gtkplus``` will fail unless your build host has cups.h available.  It seems the build process for gtkplus has a hidden dependency on cups even if the ~cups variant is the default.  
My build logs are here: https://gist.github.com/bgeltz/d33f9198a74ff0f0047766ab5ec7df02

Alternatively, I can try to build a newer version of gtkplus than 3.24.29 to see if it is resolved, but I had not yet gone down that path.  I see on gtk.org their "latest old stable" is 3.24.39.

---

Let's separate that out into a separate PR. Changing the default doesn't fix the issue, it just hides it.

---

Done.  It's here: https://github.com/spack/spack/pull/42301